jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/util/GenericErrorSummaryCountCell.java

```diff
@@ -82,8 +87,9 @@ public void render(Context context,
         sb.appendHtmlConstant(tooltip.getOuterHTML());
     }
 
-    public void openErrorView(final String pid) {
-        viewPresenter.openErrorView(pid);
+    public void openErrorView(final String pid){
+        viewPresenter.openView(pid);
```

Comment:
If you reuse _openView_ in here you will get an error list filtered by process instance id, when I believe we actually want is to filter by the task id,